### DISCUSSION TITLE
golang support in micros

### DISCUSSION THREAD

**Discussion by IndrajeethY** [deta/cloud-docs#580] [Help] [answered]
does micros support golang apps?

**Comment by jnsougata** [comment, marked as answer]
Micros on deta cloud doesn't support GoLang but Micros on deta space supports GoLang.
You can apply [here](https://form.typeform.com/to/zWD18IEB?typeform-source=alpha.deta.space) for space access!

**Reply by IndrajeethY** [reply]
Great thanks
